Reqnroll.Generator/Generation/UnitTestFeatureGenerator.cs

```diff
@@ -252,9 +253,20 @@ private void SetupTestInitializeMethod(TestClassGenerationContext generationCont
                 nameof(ITestRunner.OnFeatureEndAsync));
             _codeDomHelper.MarkCodeMethodInvokeExpressionAsAwait(onFeatureEndAsyncExpression);
 
-            //if (testRunner.FeatureContext != null && !testRunner.FeatureContext.FeatureInfo.Equals(featureInfo))
-            //  await testRunner.OnFeatureEndAsync(); // finish if different
-            testInitializeMethod.Statements.Add(
+            // VB does not allow the use of the await keyword in a "finally" clause. Therefore, we have to generate code specific to
+            // the language. The C# code will await OnFeatureStartAsync() while VB will call is sync and assign a variable to the resulting Task.
```

Comment:
Suggest minor change to the comment to clarify meaning:
The C# code will await OnFeatureStartAsync() while the VB code will call it synchronously and assign a variable to the resulting Task.

Reply:
I'm not fully sure if this VB structure work at all. If the try block will throw an exception, the finally will save the Task (and runs the sync part of it up until the first async op) but then the exception will be propagated, so the await on the task variable will never run. Or do I miss something?

Reply:
I have improved the comment.